wca-regulations.md

```diff
@@ -358,13 +358,13 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 ## <article-A><speedsolving><speedsolving> Article A: Speed Solving
 
 - A1) Speed Solving attempts must abide by the following procedures.
-    - A1a) The organization team must enforce time limits for solves and/or rounds.
-        - A1a1) The default time limit per solve is 10 minutes, though the organization team may announce a higher or lower time limit.
-        - A1a2) Cumulative time limits may be enforced (e.g. 3 solves with a cumulative time limit of 20 minutes - see [Regulation A1a5](regulations:regulation:A1a5)). The time elapsed in a DNF result counts towards the cumulative time limit.
-        - A1a3) For each round, any time limits must be announced before the round starts, and should not be changed after it has begun. Changes must be made at the discretion of the WCA Delegate, who must carefully consider the fairness of the change.
-        - A1a4) The competitor must end each solve within the time limit. If a competitor reaches the time limit for a solve/round, the judge stops the solve immediately and records the result as DNF. Exception: Multiple Blindfolded Solving (see [Regulation H1b1](regulations:regulation:H1b1)).
-        - A1a5) A solve is considered to meet the time limit if and only if the final result, after any time penalties are applied, is less than the time limit. Exception: Multiple Blindfolded Solving (see [Regulation H1b1](regulations:regulation:H1b1)).
-    - A1b) If the time limit for a solve is greater than 10 minutes, a stopwatch must be used for timekeeping.
+    - A1a) The organization team applies exactly one kind of time limit (time limit per solve or cumulative time limit) to each round.
+        - A1a1) The default time limit is 10 minutes for each attempt in a round, though the organization team may announce a higher or lower time limit per attempt. In this case, the limit must be the same for all attempts in a round.
```

Comment:
I don't agree to replace "limit per solve" with "limit per attempt". Attempt = inspection + solve + signing. We are limiting only the part in the middle, while the total attempt can be longer than a limit.

Reply:
Not quite, because of penalties. If your solve has an original recorded result of 9:59.xx but you have a penalty, then the result exceeds a time limit of 10:00. We also use "attempt" where possible.

The main issue I saw with changing "solve" to "attempt" was A1a4, but I also addressed that by explicitly referring to the timer.

Reply:
I agree with @Claster since inspection and signing do not count towards the limit. I said this stuff a while back at some issue here, I believe.

Reply:
Maybe a cleanup is due?

Reply:
Maybe a cleanup is due?

Reply:
That's true, although the last part of A1a5) maybe should be modified. I'll suggest.

Reply:
See #349 and #451 

I still believe the text as of this PR doesn't make much logical sense. Attempt begins with inspection (A3a, A3b2) and finishes after signatures (A7c).

> A1a5 makes it clear that additional time like signing doesn't count towards the time limit.

This IMO matches what I'm saying. "A solve is considered to meet the time limit...", and not "An attempt is considered..."

Reply:
It seems that @Laura-O and @viroulep also preferred "attempt" in #349.

I think that "solve" is a reasonable choice, but I still think "attempt" is a little better, and consistency is the most important. I'm happy to discuss this further in another PR, but I think I'd like to leave this as landed unless more people feel strongly about changing it again.